docs/requirements.txt

```diff
@@ -0,0 +1,30 @@
+alabaster==0.7.12
+Babel==2.9.1
+breathe==4.30.0
+certifi==2021.5.30
+charset-normalizer==2.0.4
+cpplint==1.5.4
+docutils==0.16
+flake8==3.9.2
+idna==3.2
+imagesize==1.2.0
+Jinja2==3.0.1
+MarkupSafe==2.0.1
+mccabe==0.6.1
+packaging==21.0
+pycodestyle==2.7.0
+pyflakes==2.3.1
+Pygments==2.10.0
+pyparsing==2.4.7
+pytz==2021.1
+requests==2.26.0
+snowballstemmer==2.1.0
+Sphinx==4.1.2
+sphinx-rtd-theme==0.5.2
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==2.0.0
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.5
+urllib3==1.26.6
```

Comment:
For these requirements do we really want to dictate the specific versions?  In my experience we should at least make all of these `>=`, but we may want to omit the version number unless we have a specific requirement.

Reply:
This is the standard output from `pip freeze > requirements.txt` it's mostly a stability thing as all the build tools (including the service that hosts the documentation) will manually install these requirements before building, so this makes sure they always have a working setup.

Reply:
I understand why you'd want all versions defined to have a stable build version.  But in a year from now it's entirely possible that we may be requiring all users to have very outdated packages.  Furthermore, I'm not convinced we would have any dependency issues,